src/components/PlayerList/PlayerList.js

```diff
@@ -13,21 +13,22 @@ const PlayerList = ({ players }) => {
             age={player.age}
             wins={player.wins}
             losses={player.losses}
+            challenged={player.challenged}
           />
         )
       })}
     </StyledList>
   )
 }
 
-const StyledList = styled.ul`
+export const StyledList = styled.ul`
```

Comment:
why do you use `export` here?

Reply:
I use the StyledList in the ChallengeList component as well. I didn't want to just copy and paste 😉

Reply:
Depending on the scale of the project and reusability of the styles it might even make sense to just put all the styled components in their own folder and import them when needed. But I wanted to finish, so i chose the easy route...